test/JDBC/input/BABEL-2408.sql

```diff
@@ -0,0 +1,2 @@
+EXEC sp_describe_undeclared_parameters 'COMMIT TRANSACTION;'
```

Comment:
The customer doesnt seem to have an issue with 'COMMIT TRANSACTION' stmt used with sp_describe_undeclared_parameters.
It seems that sp_describe_undeclared_parameters is being used for some datatype which is not valid for SQL Server too. 
So the python API fails and then the individual query for 'COMMIT TRANSACTION' is executed.
In babelfish we throw generic unsupported error message which is intentionally transaction aborting.
Either we introduce a better error catch scenario or change the PG error code from being unsupported and then map that error code appropriately